frameworks/Java/undertow-jersey-c3p0/src/main/java/hello/PlaintextResource.java

```diff
@@ -0,0 +1,19 @@
+package hello;
+
+import javax.inject.*;
+import javax.ws.rs.*;
+import java.util.*;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
```

Comment:
Take this line out, since it doesn't seem like we're using it. Otherwise, looks good to me!